### DISCUSSION TITLE
¿Cómo construir tu buyer persona?

### DISCUSSION THREAD

**Discussion by Marco01nD** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#135] [Q&A] [answered]
# Primeros pasos <img src="https://media.giphy.com/media/aRynp8o3ep55M2duAd/giphy.gif" alt="drawing" width="60"/>

Hola posiblemente algunos ya tienen esta parte, pero para algunos que aún les cuesta encontrar la información o que no entiendan la que existe en internet les comparto el siguiente enlace <a href="https://www.hubspot.es/make-my-persona">buyer persona</a>
El cual los guiará paso a paso para la construcción del mismo.

### Cabe aclarar que el buyer persona será quien realice la demanda.

Suerte. 

Marco

**Comment by romarpla** [comment, marked as answer]
Contestado acá --> https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/131
